src/test/ui/generic/generic-impl-more-params-with-defaults.stderr

```diff
@@ -2,7 +2,7 @@ error[E0244]: wrong number of type arguments: expected at most 2, found 3
   --> $DIR/generic-impl-more-params-with-defaults.rs:23:5
    |
 LL |     Vec::<isize, Heap, bool>::new();
-   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected at most 2 type arguments
+   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected at most 2 type argument
```

Comment:
If my comment is a duplicate, I am sorry. I just want to point out that this seems not to be correct English.

Reply:
Heh, I didn't notice this, thanks!

Reply:
Oops, fixing now, thanks!

Reply:
Great :+1: There is another one below which I believe exists for the same reason.

Reply:
@Woyten: that's correct — these are both fixed in https://github.com/rust-lang/rust/pull/53600, thanks!